app/src/lib/stores/app-store.ts

```diff
@@ -5694,6 +5698,84 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     this.emitUpdate()
   }
+
+  /** This shouldn't be called directly. See `Dispatcher`. */
+  public _initializeCherryPickProgress(
+    repository: Repository,
+    commits: ReadonlyArray<CommitOneLine>
+  ) {
+    if (commits.length === 0) {
+      // This shouldn't happen... but in case throw error.
+      throw new Error(
+        'Unable to initialize cherry pick progress. No commits provided.'
+      )
+    }
+
+    this.repositoryStateCache.updateCherryPickState(repository, () => {
+      return {
+        progress: {
+          kind: 'cherryPick',
+          title: `Cherry picking commit 1 of ${commits.length} commits`,
+          value: round(0, 2),
```

Comment:
Is there any specific reason for using `round(0, 2)` instead of just `0`? 🤔 
```suggestion
          value: 0,
```

Reply:
Probably not... that came to mind to me as well and never went back and tried it as just 0.. Rebase did this so I followed suit.. I wondered if for some reason we needed 0.00 .. I will try it without.

Reply:
Seems to work all the same. I removed it.